data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
+			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
+			choice
+				`	"So, what is our next step?"`
+					goto labnext
+				`	"Seeker?"`
+			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
+			choice
+				`	"I thought all Remnant were spacefarers?"`
+				`	"Okay. So what next?"`
+					goto labnext
+			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
+			choice
+				`	"Years drifting in the void?"`
+				`	"Okay, let's get on with this."`
+					goto labnext
+			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
+			label labnext
+			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a jump drive you can take. Meet me in the starport once you have a jump drive."`
+
+
+
+event "remnant: lab availability"
+	outfitter "Remnant"
+		add "Research Laboratory"
+
+
+
+mission "Remnant: Expanded Horizons Astral 2"
+	name "Research expedition to Sagittarius A*"
+	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."
+	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."
+	passengers 1
```

Comment:
Okay, so if the outfit takes up a bunk, then isn't the person taking up that bunk Dawn (thus rendering the passenger requirement here unnecessary)? Or is there a separate scientist traveling with you?

Reply:
Hmmm... For the sake of the story, I think I'm going to tweak the requirements to say that it is Dawn taking up that spot.

In the future, though, that will be a generic "science officer" or whatever you prefer to call them.

Reply:
Made the adjustment to the passenger requirement (removing it from Astral 2), and added a comment in the conclusion from Dawn that she ran it for this mission, but if the player keeps it installed their ship will automatically hire a scientist to run it.